src/customwidgets/notesdockwidget.cpp

```diff
@@ -69,7 +67,7 @@ void NotesDockWidget::removeNoteRow()
         rows << idx.row();
     }
 
-    qSort(rows.begin(), rows.end(), qGreater<int>());
+    std::sort(rows.begin(), rows.end(), std::greater<int>());
```

Comment:
Why use std instead of Qt version?

Reply:
The Qt ones are deprecated in Qt5:

[source](https://doc.qt.io/qt-5/qtalgorithms.html)

> # Qt and the STL Algorithms
> Historically, Qt used to provide functions which were direct equivalents of many STL algorithmic functions. Starting with Qt 5.0, you are instead encouraged to use directly the implementations available in the STL; most of the Qt ones have been deprecated (although they are still available to keep the old code compiling).